robot/game.py

```diff
@@ -3,7 +3,7 @@
 from typing import NewType
 from robot.board import Board
 
-Zone = NewType('Zone', int)
+TZone = NewType('TZone', int)
```

Comment:
Ah, apologies. My comment about naming types like this was intended to refer only to generic type variable types -- since the type isn't concrete there's a desire to show that.

For concrete types (even when they're newly defined types), I think we should stick with just having a plain type name.

Reply:
Resolved in #59 by 54becc903d1bf9259648f59c77f48fff5642e107.